README.md

```diff
@@ -34,53 +46,35 @@ A LocalizedText object cannot be used in FHIR-conforming types due to its incomp
 }
 ```
 
-## /invitations/$invitationId$
+## invitations/$invitationId$
 
 When a patient joins Engage-HF, we first create an invitation code on demand of an operator of the web dashboard. Upon activation, we create a user associated with the invitationCode and remove the entry in this collection.
 
 |Property|Type|Values|Comments|
 |-|-|-|-|
 |used|boolean|true, false|Whether the invitation code has already been used or not. If this property is true, the invitation code can no longer be used.|
 |usedBy|optional string|-|The user created from this invitation code.|
+|auth|optional Auth|Authentication information to be set when redeeming invitation.Will be undefined once invitation has been redeemed.|
```

Comment:
Wondering if undefined is the right word here? Maybe not present/null?

Reply:
Changed to `Will no longer be set once invitation has been redeemed.`